cmd/kots/cli/install.go

```diff
@@ -328,6 +328,10 @@ func InstallCmd() *cobra.Command {
 				log.ActionWithoutSpinner("")
 				log.ActionWithoutSpinner("To access the Admin Console, run kubectl kots admin-console --namespace %s", namespace)
 				log.ActionWithoutSpinner("")
+			} else {
+				log.ActionWithoutSpinner("")
+				log.ActionWithoutSpinner("Done")
```

Comment:
Without this, the last thing printed is an ambiguous error, making it look like install failed.

Reply:
Why is that EOF logged?